packages/application-generic/src/usecases/compile-template/compile-template.spec.ts

```diff
@@ -1,22 +1,17 @@
 import { Test } from '@nestjs/testing';
-import { UserSession } from '@novu/testing';
 
 import { CompileTemplate } from './compile-template.usecase';
 import { CompileTemplateCommand } from './compile-template.command';
 
 describe('Compile Template', function () {
   let useCase: CompileTemplate;
-  let session: UserSession;
 
   beforeEach(async () => {
     const moduleRef = await Test.createTestingModule({
-      imports: [CompileTemplate],
-      providers: [],
+      imports: [],
+      providers: [CompileTemplate],
```

Comment:
It's impossible to compile this testing module, because you cannot import provider (class with `@Injectable` decorator) - you can only import modules - classes with `@Module` decorator.

I have no idea how these tests worked before. Maybe I am doing something wrong, but there is no documentation how to run tests inside `application-generic`.